includes/ucf-section-common.php

```diff
@@ -442,7 +442,13 @@ public static function add_sections_to_post() {
 				'scripts' => array()
 			);
 
-			if ( count( $object->sections['posts'] ) < 1 ) return $object;
+			if ( count( $object->sections['posts'] ) < 1 ) {
+				if( $cta_section = get_field( 'college_page_cta_section', $object ) ) {
+					$object->sections['posts'] = array( $cta_section );
+				} else {
+					return $object;
+				}
```

Comment:
It's a little confusing having this logic nested here. It might read a little easier if the check for the college_page cta field is done prior to the count check. At that point, if there is a section for that field, the count will be >= 1 and the logic can continue as normal.